### PR TITLE
fix link in README.md

### PR DESCRIPTION
**Pull request by luism3861**
This PR fix link in README.md when you click appears 404 error.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ request to Vercel. (We use Vercel because of their nice GitHub integration.)
 ## History
 
 This is a rewrite of the Deno website it will combine the code in
-https://github.com/denoland/deno/tree/master/website and
+https://github.com/denoland/deno/tree/master and
```
Comment (kidonng) [inline README.md:26]: It is better to use the available old tree instead:

```diff
- https://github.com/denoland/deno/tree/master/website
+ https://github.com/denoland/deno/tree/f96aaa802b245c8b3aeb5d57b031f8a55bb07de2
```

Comment (luism3861) [reply README.md]: thanks i change your suggestion  :) !

**Comment by lucacasonato** [comment]
@luism3861 Can you merge with master to get the CI green?

**Comment by luism3861** [comment]
> @luism3861 Can you merge with master to get the CI green?

can i create a new PR i can't merge to master?

**Comment by luism3861** [comment]
> @luism3861 Can you merge with master to get the CI green?



> > @luism3861 Can you merge with master to get the CI green?
> 
> can i create a new PR i can't merge to master?

the problem is in vercel need authorization to deploy

**Comment by luism3861** [comment]
> @luism3861 Can you merge with master to get the CI green?

 @lucacasonato  ci its succesful  in web and worker the problem is in vercel need authorization to deploy ?

**Comment by lucacasonato** [comment]
@luism3861 Just run ‘git merge origin master’ inside of you local copy of this repository.

**Comment by luism3861** [comment]
> @luism3861 Just run ‘git merge origin master’ inside of you local copy of this repository.

i try to use git merger origin master but appears everything-to-date , @lucacasonato could  i create a new PR and send this again?

**Comment by ry** [comment]
Thanks for the patch, but it must be green to be considered. Due to the high volume, I'm not reviewing red PRs, just closing to clear out the queue. I apologize if I haven't addressed something. Please open a new PR.

**Pull request closed without merging**